testonly/loglb/main.go

```diff
@@ -65,6 +66,7 @@ func newRandomLoadBalancer(serverCfg string) (*randomLoadBalancer, error) {
 	return &lb, nil
 }
 
+// TODO(somebody): This function name collides with a builtin function and should be renamed
```

Comment:
Didn't realize it would be controversial as I usually call lifecycle methods shutdown(). As you obviously care more about this than I do (e.g. at all) I'll take it out.